ConsumerApi.Tests.Integration/StepDefinitions/RelationshipsStepDefinitions.cs

```diff
@@ -0,0 +1,196 @@
+
+        if (_rejectRelationshipChangeResponse != null)
+            ((int)_rejectRelationshipChangeResponse!.Status).Should().Be(expectedStatusCode);
+
+        if (_revokeRelationshipChangeResponse != null)
+            ((int)_revokeRelationshipChangeResponse!.Status).Should().Be(expectedStatusCode);
+    }
+
+    [Then(@"the response content contains an error with the error code ""([^""]*)""")]
+    public void ThenTheResponseContentIncludesAnErrorWithTheErrorCode(string errorCode)
+    {
+        if (_createRelationshipResponse != null)
+        {
+            _createRelationshipResponse!.Error.Should().NotBeNull();
+            _createRelationshipResponse.Error!.Code.Should().Be(errorCode);
+        }
+
+        if (_acceptRelationshipChangeResponse != null)
+        {
+            _acceptRelationshipChangeResponse!.Error.Should().NotBeNull();
+            _acceptRelationshipChangeResponse.Error!.Code.Should().Be(errorCode);
+        }
+
+        if (_rejectRelationshipChangeResponse != null)
+        {
+            _rejectRelationshipChangeResponse!.Error.Should().NotBeNull();
+            _rejectRelationshipChangeResponse.Error!.Code.Should().Be(errorCode);
+        }
+
+        if (_revokeRelationshipChangeResponse != null)
+        {
+            _revokeRelationshipChangeResponse!.Error.Should().NotBeNull();
+            _revokeRelationshipChangeResponse.Error!.Code.Should().Be(errorCode);
+        }
+    }
+
+    [Then("the response contains a Relationship")]
+    public void ThenTheResponseContainsARelationship()
+    {
+        _createRelationshipResponse!.Should().BeASuccess();
+        _createRelationshipResponse!.Should().ComplyWithSchema();
+    }
+
+    [Then("the response contains an AcceptRelationshipChangeResponse")]
+    public void ThenTheResponseContainsAnAcceptRelationshipChangeResponse()
+    {
+        _acceptRelationshipChangeResponse!.Should().BeASuccess();
+        _acceptRelationshipChangeResponse!.Should().ComplyWithSchema();
+    }
+
+    [Then("the response contains an RejectRelationshipChangeResponse")]
+    public void ThenTheResponseContainsAnRejectRelationshipChangeResponse()
+    {
+        _rejectRelationshipChangeResponse!.Should().BeASuccess();
+        _rejectRelationshipChangeResponse!.Should().ComplyWithSchema();
+    }
+
+    [Then("the response contains an RevokeRelationshipChangeResponse")]
+    public void ThenTheResponseContainsAnRevokeRelationshipChangeResponse()
+    {
+        _revokeRelationshipChangeResponse!.Should().BeASuccess();
+        _revokeRelationshipChangeResponse!.Should().ComplyWithSchema();
+    }
+
+    private async Task<ApiResponse<CreateRelationshipTemplateResponse>> CreateRelationshipTemplate(Client client)
+    {
+        var createRelationshipTemplateRequest = new CreateRelationshipTemplateRequest
+        {
+            Content = ConvertibleString.FromUtf8("AAA").BytesRepresentation
```

Comment:
I start to see this `ConvertibleString.FromUtf8("AAA").BytesRepresentation` more and more often. But ConvertibleString wasn't created for this purpose. 
I'd suggest an extension method that allows us to do `"AAA".GetBytes()`.
After creating it, use it everywhere please.